x-pack/plugin/enrich/qa/common/src/main/java/org/elasticsearch/test/enrich/CommonEnrichRestTestCase.java

```diff
@@ -129,6 +129,16 @@ public void testDeleteExistingPipeline() throws Exception {
         assertOK(client().performRequest(getRequest));
     }
 
+    public static String generatePolicySource(String index) throws IOException {
```

Comment:
nit: can you add a radom `query` field with a match all ? (I know the behavior is the same but will help catch a regression since we rarely (ever?) test parsing the `query` field)